projections/shapes/sig/shapes/client.rbs

```diff
@@ -4,8 +4,7 @@ module ShapeService
 
     def self.new: (
       ?adaptive_retry_wait_to_fill: bool,
-      ?auth_resolver: ShapeService::AuthResolver,
-      ?auth_schemes: Hash[String, Smithy::Client::AuthScheme],
+      ?auth_resolver: Smithy::Client::AuthResolver,
```

Comment:
This should be ShapeService::AuthResolver somehow. I don't believe we have an AuthResolver base class in Smithy Client.